src/typings.d.ts

```diff
@@ -2,12 +2,12 @@ declare module 'express-http-proxy';
 
 declare module 'express-robots-txt';
 
-declare var PRODUCTION_MODE: boolean;
+declare let PRODUCTION_MODE: boolean;
```

Comment:
In #1080 I changed these to const, which makes more sense in my opinion.

Reply:
So, changed to 'const' which is even better. ;)
Thanks for the feedback.